static/components/PoiCard.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+
+class PoiCard extends React.Component {
+
+  render() {
+    const distance = distance => {
```

Comment:
I would move this out of `render()`; which is a method that should be focused on rendering, meaning calculating visual stuff, here DOM/HTML.
Maybe could you create a method like `calculateDistance(from, to)` that `render()` would call.